### PR TITLE
[Importer] Support GlobalAveragePool op in ONNX importer

### PR DESCRIPTION
**Pull request by beicy**
GobalAveragePool operator is supported in ONNX. Add this one in ONNX loader

### CODE REVIEW

File: lib/Importer/ONNX.cpp [hunk 1]
```diff
@@ -379,6 +379,24 @@ void ONNXModelLoader::loadOperator(const onnx::NodeProto &op) {
     return;
   }
 
+  if (typeName == "GlobalAveragePool") {
+    // Load the inputs:
+    auto *in = getOrCreateNodeByName(op.input(0));
+    int stride =
+      dict.count("strides") ? getConstantArrayHead(dict["strides"]) : 1;
+    size_t kernel = in->dims()[2];
```
Comment (artemrakhov) [inline lib/Importer/ONNX.cpp]: let's assert that height == width

File: lib/Importer/ONNX.cpp [hunk 2]
```diff
@@ -379,6 +379,24 @@ void ONNXModelLoader::loadOperator(const onnx::NodeProto &op) {
...
+    int pad = getPad(dict);
+    auto *tr = G_.createTranspose(opName, in, NCHW2NHWC);
+    Node *node = nullptr;
```
Comment (artemrakhov) [inline lib/Importer/ONNX.cpp]: `Node *node = G_.createPoolAvg(opName, tr, kernel, stride, pad);`

File: tests/images/run.sh [hunk 3]
```diff
@@ -11,5 +11,8 @@ done
 for png_filename in tests/images/imagenet/*.png; do
   ./bin/loader $png_filename -image_mode=128to127 -m=vgg19/model.onnx $@
 done
+for png_filename in tests/images/imagenet/*.png; do
+    ./bin/loader $png_filename -image_mode=128to127 -m=squeezenet/model.onnx $@
```
Comment (artemrakhov) [inline tests/images/run.sh]: nit: please remove 2 spaces to align with other fors

**Comment by nadavrot** [comment]
Nice!

**Pull request merged**